docker-compose.yml.example

```diff
@@ -4,17 +4,64 @@ services:
     image: ghcr.io/unownhash/golbat:main
     container_name: golbat
     restart: unless-stopped
+    ports:
+      - "9001:9001"
     volumes:
-      - type: bind
-        source: ./config.toml
-        target: /golbat/config.toml
+      # EITHER you mount the config.toml file
+      # OR you use environment variables below
+      - ${PWD}/config.toml:/golbat/config.toml
       # optional volumes if you want to have access to logs and persist cache
       # comment if not needed
-      - type: volume
-        source: ${PWD}/cache
-        target: /golbat/cache
-      - type: volume
-        source: ${PWD}/logs
-        target: /golbat/logs
-    ports:
-      - "9001:9001"
+      - ${PWD}/cache:/golbat/cache
+      - ${PWD}/logs:/golbat/logs
+    environment:
+      # refer to config.toml.example and README for explanations
+      - GOLBAT.PORT=9001
+      - GOLBAT.RAW_BEARER=
+      - GOLBAT.API_SECRET=golbat
+
+      - GOLBAT.STATS=false
+      - GOLBAT.IN_MEMORY=false
+
+      - GOLBAT.KOJI.URL=http://{koji_url}/api/v1/geofence/feature-collection/{golbat_project}
+      - GOLBAT.KOJI.BEARER_TOKEN=secret
+
+      - GOLBAT.CLEANUP.POKEMON=true
+      - GOLBAT.CLEANUP.INCIDENTS=true
+      - GOLBAT.CLEANUP.QUESTS=true
+      - GOLBAT.CLEANUP.STATS=true
+      - GOLBAT.CLEANUP.STATS_DAYS=7
+
+      - GOLBAT.LOGGING.DEBUG=false
+      - GOLBAT.LOGGING.SAVE.LOGS=true
+
+      - GOLBAT.DATABASE.USER=
+      - GOLBAT.DATABASE.PASSWORD=
+      - GOLBAT.DATABASE.ADDRESS=host.docker.internal:3306
+      - GOLBAT.DATABASE.DB=
+
+      - GOLBAT.PVP.ENABLED=true
+      - GOLBAT.PVP.INCLUDE_HUNDOS_UNDER_CAP=false
+      - GOLBAT.PVP.LEVEL_CAPS=50,51
+
+      - GOLBAT.PVP.LEAGUES.0.NAME=little
```

Comment:
I wonder whether we should take the opportunity to tidy up pvp configuration as these are just trash, especially here in docker config. We could do as a seperate PR (I would be happy to do first and you merge up?) or we could do as part of this one

Reply:
You mean setting default values or removing the config altogether?

Reply:
as per malte's issue discussion (or small variation) - getting rid of the config altogether. The leagues are the leagues, the only real option is whether the 500 lead has the evolution restriction (and the global settings for level caps etc, for which there would be sensible defaults)